[9:08 PM] Lokesh Kannaiah
Requirement already satisfied: asgiref==3.7.2 in c:\program files\python312\lib\site-packages (from -r .\requirments.txt. (line 2)) (3.7.2)

WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationErr

or(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1000)'))': /simple/babel/

WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationErr

or(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1000)'))': /simple/babel/

WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationErr

or(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1000)'))': /simple/babel/

WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationErr

or(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1000)'))': /simple/babel/

WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationErr

or(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1000)'))': /simple/babel/

Could not fetch URL https://pypi.org/simple/babel/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=4

43): Max retries exceeded with url: /simple/babel/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate ve

rify failed: self-signed certificate in certificate chain (_ssl.c:1000)'))) - skipping

ERROR: Could not find a version that satisfies the requirement Babel==2.14.0 (from versions: none)

ERROR: No matching distribution found for Babel==2.14.0

Could not fetch URL https://pypi.org/simple/pip/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443

): Max retries exceeded with url: /simple/pip/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify

failed: self-signed certificate in certificate chain (_ssl.c:1000)'))) - skipping

PS C:\Users\Administrator\success-pilot-web\success-pilot-web>ReadMe File
